apps/govquests-api/rails_app/db/seeds.rb

```diff
@@ -70,9 +70,23 @@ def create_action(action_data)
     min_score: 20
   }
 }
+
+verify_position_action = {
+  action_type: "verify_position",
+  display_data: {
+    title: "Verify your position",
+    description: "Click to verify your status as a Top 100 Delegate."
+  },
+  action_data: {
+    action_type: "verify_position",
+    min_position: 100
```

Comment:
this wasn't too clear to me. you could perhaps rename it to sth like `should_be_higher_than`

Reply:
You're right, I was not happy with the name too, I'll change it!